src/redux/books/books.js

```diff
@@ -46,15 +46,15 @@ export const fetchBooks = () => async (dispatch) => {
 };
 export const removeBook = (id) => async (dispatch) => {
   dispatch(remove(id));
-  await fetch(`https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/HmsXokvT4r0o4d4gM6bR/books/${id}`, {
+  await fetch(`https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/SERz0T6MOlfncfi0umcc/books/${id}`, {
     method: 'DELETE',
     body: JSON.stringify({ item_id: id }),
     headers: { 'Content-type': 'application/JSON' },
   });
 };
 export const addBookToAPI = (payload) => async (dispatch) => {
   dispatch(addBook(payload));
-  await fetch('https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/HmsXokvT4r0o4d4gM6bR/books', {
+  await fetch('https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/SERz0T6MOlfncfi0umcc/books', {
     method: 'POST',
     body: JSON.stringify(payload),
     headers: { 'Content-type': 'application/JSON' },
```

Comment:
The ` addBookToAPI` action creator is not working. When I try to add a new book, while the `state` is updated with the new book correctly, the new book however is not posted to the API. 
I receive a POST error on the console, and when I check on postman using your id the JSON is empty.

![image](https://user-images.githubusercontent.com/38088818/166976724-c0ffce18-008a-4c82-a230-2d684db8cdfd.png)

- Please click on the image to see the comments clearly.